### PR TITLE
CSE Machine: Display implicit block for program environment in the Control

### PR DESCRIPTION
**Pull request by DiligentPenguinn**
### Description
Currently, there are no paranthesis wrapping the program when first pushed onto the control

image

This PR aims to fix this. Expected outcome:

image

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/features/envVisualizer/EnvVisualizerUtils.ts [hunk 1]
```diff
@@ -425,7 +442,10 @@ export function getControlItemComponent(
           topItem
         );
       default:
+        console.log("Im not a program");
+        console.log("Control item: ", controlItem);
         const text = astToString(controlItem).trim();
+        console.log("text: ", text);
```
Comment (RichDom2185) [inline src/features/envVisualizer/EnvVisualizerUtils.ts:448]: Could we remove the console.log statements?

**Review by RichDom2185 — COMMENTED** [review]
Hi, thanks for working on this, please run `yarn format` and commit the changes to ensure the files are properly formatted, thanks!

File: src/features/envVisualizer/EnvVisualizerUtils.ts (hunk 1 above)
Comment (DiligentPenguinn) [reply src/features/envVisualizer/EnvVisualizerUtils.ts]: Oops I forgot to remove these, I will format the files again as well

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**